### DISCUSSION TITLE
Fix number of decimal places

### DISCUSSION THREAD

**Discussion by moritzdingerheraeus** [pimcore/pimcore#9374] [Ideas]
We require an addition that allows use to specify the number of decimal places for numeric fields via the API.
Eg. we would like to output 1 as 1.0, or 2 as 2.0.

**Comment by weisswurstkanone** [comment]
Why not implement this on the client side, feels like a pure output issue.

**Comment by moritzdingerheraeus** [comment]
This could be done on client side but I rather see that as logic that the API should handle and client would only display.
Handling in the client is especially cumbersome if different rules for different attributes should apply.

**Comment by weisswurstkanone** [comment]
can you provide a PR then please?

**Comment by weisswurstkanone** [comment]
@moritzdingerheraeus I wanna move this to the discussion forum for now. Please feel free to open a PR at any time. Thx!
